### PR TITLE
Horizontal My Stuff tabs addon

### LINKED ISSUE #4196: Option to change scratchr2 navigation layout
### Is your request related to a problem? Please describe.

Not everyone likes the my stuff or settings navigation layout after the `scratchr2` addon update.

### Describe the solution you'd like

Adding a toggle to `scratchr2` to move the navigation to the top of stuff or settings like it was before 1.24.

### Describe alternatives you've considered

Removing the new navigation completely or making it a new addon.

### PR DESCRIPTION
**Pull request by mxmou**
Resolves #4196

### Changes

Adds an addon that makes the tab bar in My Stuff horizontal.
Without scratchr2 addon
With scratchr2 addon

Enabling or disabling 2.0 → 3.0 currently breaks it (#4188).

### CODE REVIEW

**Comment by Norbiros** [comment]
❤️  I love that addon!

**Comment by WorldLanguages** [comment]
> Enabling or disabling 2.0 → 3.0 currently breaks it (#4188).

Fixed by #4289

File: addons/horizontal-mystuff-tabs/addon.json [hunk 1]
```diff
@@ -0,0 +1,28 @@
+{
+  "name": "Horizontal tabs on My Stuff",
+  "description": "Moves the My Stuff navigation buttons (\"All Projects\", \"My Studios\", \"Trash\") to the top of the page, instead of keeping them on the left.",
+  "tags": ["community"],
```
Comment (WorldLanguages) [inline addons/horizontal-mystuff-tabs/addon.json:4]: @mxmou Any reason this shouldn't be a theme?

**Comment by Samq64** [comment]
Should this addon also affect Account settings?

**Comment by WorldLanguages** [comment]
> Should this addon also affect Account settings?

See https://github.com/ScratchAddons/ScratchAddons/issues/4196#issuecomment-1030779787

**Comment by RedGuy12** [comment]
I think it would be good even though the pills are on the other side

File: addons/horizontal-mystuff-tabs/addon.json (hunk 1 above)
Comment (mxmou) [reply addons/horizontal-mystuff-tabs/addon.json:4]: It could be - editor-stage-left has a similar purpose and is a theme.

**Comment by mxmou** [comment]
> Should this addon also affect Account settings?

This addon is mainly intended for people who are used to the previous behavior of the Scratch 2.0 → 3.0 addon, which moved My Stuff tabs to the top and Account Settings tabs to the right. There is no good reason to put tabs on the right, so this addon only reverts the My Stuff change. It could also provide an option to make the Account Settings tabs horizontal, but it should probably be disabled by default.

**Pull request merged**